src/components/organisms/PaymentModal/PaymentModal.tsx

```diff
@@ -1,93 +0,0 @@
-import React, { useState } from "react";
-import { Modal } from "react-bootstrap";
-
-import { AnyVenue, VenueEvent } from "types/venues";
-
-import useConnectUserPurchaseHistory from "hooks/useConnectUserPurchaseHistory";
-import { hasUserBoughtTicketForEvent } from "utils/hasUserBoughtTicket";
-import { isUserAMember } from "utils/isUserAMember";
-import { Purchase } from "types/Purchase";
-import "./PaymentModal.scss";
-import PaymentForm from "./PaymentForm";
-import PaymentConfirmation from "./PaymentConfirmation";
```

Comment:
Have we removed all of these corresponding files from the codebase? If not, we almost certainly should be. I know some of them still exist for sure, yet they almost certainly shouldn't, and thus imply there is more code that needs to be cleaned up here still.

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40